controller/api/models.py

```diff
@@ -410,6 +410,19 @@ def _has_hostname(image):
         escaped_command = command.replace("'", "'\\''")
         return c.run(escaped_command)
 
+    def maintenance_on(self, user):
+        msg = "{} enables miantenance mode".format(user.username)
+        log_event(self, msg)
+        _etcd_client.write('/deis/maintenance/{}'.format(id), True)
+
+    def maintenance_on(self, user):
```

Comment:
yeah I noticed as well. As said, thiis was done in 20m or so.

I was more intrested in general remarks of setting an etc key, and then using that in the router is a good way forward.
Also I wanted people to know I was onto this, to not do double work